src/components/box/Box.js

```diff
@@ -30,17 +31,45 @@ export const BoxDownload = ({ files, isLoading, showSizeWarning }) => (
   </Box>
 )
 
-export const RawBoxAdd = ({ files, isLoading, shareLink, t }) => (
-  <Box>
-    <AddFiles />
-    { isLoading && <Loader /> }
-    <FileTree files={files} />
-    { shareLink && <CopyLink shareLink={shareLink} /> }
-    <div className='f7 gray lh-copy'>
-      {t('box.footNote')}
+export const RawBoxAdd = ({ files, isLoading, shareLink, doAddFiles, t }) => {
```

Comment:
```suggestion
const RawBoxAddComponent = ({ files, isLoading, shareLink, doAddFiles, t }) => {
// ...
}

export const RawBoxAdd = connect(
    'doAddFiles',
    RawBoxAddComponent
)
```

Reply:
In this case, what we want is to  merge with existing `BoxAdd` component, since it uses `withTranslation`